### PR TITLE
update search and observability plugins

### PR DESCRIPTION
**Pull request by anirudha**
update search and observability plugins for 1.2
Signed-off-by: Anirudha Jadhav jadhanir@amazon.com
### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by dblock** [comment]
The DCO check failed, needs a -s amend.

File: manifests/1.2.0/opensearch-dashboards-1.2.0.yml [hunk 1]
```diff
@@ -16,4 +16,19 @@ components:
 - name: securityDashboards
   repository: https://github.com/opensearch-project/security-dashboards-plugin.git
   ref: "main"
+- name: queryWorkbenchDashboards
+  repository: https://github.com/opensearch-project/sql.git
+  working_directory: workbench
+  ref: "1.2"
```
Comment (kavilla) [inline manifests/1.2.0/opensearch-dashboards-1.2.0.yml:22]: `ref` refers to the git branch. It will fail if the "1.2" branch doesn't exist so most likely this will be `main` until the repo cuts a `1.2` branch. Also there is a PR to bump this up: https://github.com/opensearch-project/sql/pull/254. This PR will need to be merged before we can add it to the manifest.

File: manifests/1.2.0/opensearch-dashboards-1.2.0.yml [hunk 2]
```diff
@@ -16,4 +16,19 @@ components:
...
+- name: reportsDashboards
+  repository: https://github.com/opensearch-project/dashboards-reports.git
+  working_directory: dashboards-reports
+  ref: "1.2"
```
Comment (kavilla) [inline manifests/1.2.0/opensearch-dashboards-1.2.0.yml:26]: Same as above but: https://github.com/opensearch-project/dashboards-reports/pull/203

File: manifests/1.2.0/opensearch-dashboards-1.2.0.yml [hunk 3]
```diff
@@ -16,4 +16,19 @@ components:
...
+- name: traceAnalyticsDashboards
+  repository: https://github.com/opensearch-project/trace-analytics.git
+  ref: "1.2"
```
Comment (kavilla) [inline manifests/1.2.0/opensearch-dashboards-1.2.0.yml:29]: Same as above but: https://github.com/opensearch-project/trace-analytics/pull/189

File: manifests/1.2.0/opensearch-dashboards-1.2.0.yml [hunk 4]
```diff
@@ -16,4 +16,19 @@ components:
...
+- name: ganttChartDashboards
+  repository: https://github.com/opensearch-project/dashboards-visualizations
+  working_directory: gantt-chart
+  ref: "1.2"
```
Comment (kavilla) [inline manifests/1.2.0/opensearch-dashboards-1.2.0.yml:33]: This was is all good! `1.2` branch exists and plugin is bumped! If anything it might be easiest to pull this one into another PR so that it's not blocked waiting on the other PRs.

**Review by kavilla — CHANGES_REQUESTED** [review]
Blocked on other PRs. Also will need to rebase since manifest was updated.

NIT: Commit message could be a little bit more descriptive.

**Comment by joshuali925** [comment]
branch is in this repo I can't amend, i'll open another PR

**Comment by dblock** [comment]
@anirudha Please make branches off your fork.
